### PR TITLE
Connectedcache live test

### PR DESCRIPTION
**Pull request by jliusan**
add live test

### CODE REVIEW

File: sdk/resourcemanager/connectedcache/armconnectedcache/connectedcache_live_test.go [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+		log.Fatalf("failed to obtain a credential: %v", err)
+	}
+	clientFactory, err := armconnectedcache.NewClientFactory(recording.GetEnvVariable("AZURE_SUBSCRIPTION_ID", "00000000-0000-0000-0000-000000000000"), cred, testsuite.options)
+	if err != nil {
+		log.Fatalf("failed to create client: %v", err)
+	}
+	poller, err := clientFactory.NewIspCustomersClient().BeginCreateOrUpdate(testsuite.ctx, testsuite.resourceGroupName, "MccRPTest2", armconnectedcache.IspCustomerResource{
+		Location: to.Ptr(testsuite.location),
+		Properties: &armconnectedcache.CustomerProperty{
+			// Customer: &armconnectedcache.CustomerEntity{
```
Comment (tadelesh) [inline sdk/resourcemanager/connectedcache/armconnectedcache/connectedcache_live_test.go:97]: do not leave commented code in test.

Comment (jliusan) [reply sdk/resourcemanager/connectedcache/armconnectedcache/connectedcache_live_test.go:97]: I have removed  commented code

Comment (tadelesh) [reply sdk/resourcemanager/connectedcache/armconnectedcache/connectedcache_live_test.go:97]: still exist.

File: sdk/resourcemanager/connectedcache/armconnectedcache/connectedcache_live_test.go [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+	if err != nil {
+		log.Fatalf("failed to create client: %v", err)
+	}
+	IspCustomersClientDeleteResponsePoller, err := clientFactory.NewIspCustomersClient().BeginDelete(testsuite.ctx, testsuite.resourceGroupName, "MccRPTest2", nil)
+	testsuite.Require().NoError(err)
+	_, err = testutil.PollForTest(testsuite.ctx, IspCustomersClientDeleteResponsePoller)
+	testsuite.Require().NoError(err)
+}
+
+func (testsuite *ConnectedCacheTestSuite) TestCreateIspCustomersClient() {
```
Comment (tadelesh) [inline sdk/resourcemanager/connectedcache/armconnectedcache/connectedcache_live_test.go:85]:
```suggestion
func (testsuite *ConnectedCacheTestSuite) TestIspCustomersCreateOrUpdate() {
```

File: sdk/resourcemanager/connectedcache/armconnectedcache/connectedcache_live_test.go [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+	}, nil)
+	if err != nil {
+		log.Fatalf("failed to finish the request: %v", err)
+	}
+	_, err = testutil.PollForTest(testsuite.ctx, poller)
+	testsuite.Require().NoError(err)
+	fmt.Println("finish TestCreateIspCustomersClient====")
+}
+
+func (testsuite *ConnectedCacheTestSuite) TestGetIspCustomersClient() {
```
Comment (tadelesh) [inline sdk/resourcemanager/connectedcache/armconnectedcache/connectedcache_live_test.go:140]:
```suggestion
func (testsuite *ConnectedCacheTestSuite) TestIspCustomersGet() {
```

**Review by tadelesh — APPROVED** [review]

File: sdk/resourcemanager/connectedcache/armconnectedcache/connectedcache_live_test.go (hunk 3 above)
Comment (jliusan) [reply sdk/resourcemanager/connectedcache/armconnectedcache/connectedcache_live_test.go]: renamed

File: sdk/resourcemanager/connectedcache/armconnectedcache/connectedcache_live_test.go (hunk 2 above)
Comment (jliusan) [reply sdk/resourcemanager/connectedcache/armconnectedcache/connectedcache_live_test.go]: renamed

**Pull request merged**